committee-steering/governance/README.md

```diff
@@ -1,23 +1,26 @@
-# SIG Governance Template
+# SIG Charter Guide
 
-## Goals
-
-The following documents outline recommendations and requirements for SIG governance structure and provide
-template documents for SIGs to adapt.  The goals are to define the baseline needs for SIGs to self govern
-and organize in a way that addresses the needs of the core Kubernetes project.
-
-The documents are focused on:
+## Steps to create a SIG charter
 
-- Outlining organizational responsibilities
-- Outlining organizational roles
-- Outlining processes and tools
+1. Copy the template into a new file under community/sig-*YOURSIG*/charter ([sig-architecture example])
+2. Read the [Recommendations and requirements] so you have context for the template
+3. Customize your copy of the template for your SIG.  Feel free to make adjustments as needed.
+4. Update [sigs.yaml] with the individuals holding the roles as defined in the template.
```

Comment:
Completely off topic, I'm astounded that this link to sigs yaml works. HOW DOES THIS WORK

Reply:
Look at the bottom of the page